core/src/Instance/AadInstanceDiscovery.cs

```diff
@@ -84,51 +115,22 @@ public static string BuildInstanceDiscoveryEndpoint(string host)
             return string.Format(CultureInfo.InvariantCulture, "https://{0}/common/discovery/instance", host);
         }
 
-        internal async Task<InstanceDiscoveryResponse>
-            DoInstanceDiscoveryAndCacheAsync(
-                IHttpManager httpManager,
-                ITelemetryManager telemetryManager,
-                Uri authority,
-                bool validateAuthority,
-                RequestContext requestContext)
-        {
-            InstanceDiscoveryResponse discoveryResponse =
-                await SendInstanceDiscoveryRequestAsync(
-                    httpManager,
-                    telemetryManager,
-                    authority,
-                    requestContext).ConfigureAwait(false);
-
-            if (!validateAuthority)
-            {
-                Validate(discoveryResponse);
-            }
-
-            CacheInstanceDiscoveryMetadata(authority.Host, discoveryResponse);
-
-            return discoveryResponse;
-        }
-
-        private static async Task<InstanceDiscoveryResponse> SendInstanceDiscoveryRequestAsync(
-            IHttpManager httpManager,
-            ITelemetryManager telemetryManager,
+        private async Task<InstanceDiscoveryResponse> SendInstanceDiscoveryRequestAsync(
             Uri authority,
             RequestContext requestContext)
         {
-            OAuth2Client client = new OAuth2Client(httpManager, telemetryManager);
+            var client = new OAuth2Client(_httpManager, _telemetryManager);
             client.AddQueryParameter("api-version", "1.1");
             client.AddQueryParameter("authorization_endpoint", BuildAuthorizeEndpoint(authority.Host, GetTenant(authority)));
 
-            var discoveryHost = AadAuthority.IsInTrustedHostList(authority.Host) ?
-                authority.Host :
-                AadAuthority.DefaultTrustedHost;
+            string discoveryHost = AadAuthority.IsInTrustedHostList(authority.Host)
+                                       ? authority.Host
+                                       : AadAuthority.DefaultTrustedHost;
 
             string instanceDiscoveryEndpoint = BuildInstanceDiscoveryEndpoint(discoveryHost);
 
-            InstanceDiscoveryResponse discoveryResponse =
-                await
-                    client.DiscoverAadInstanceAsync(new Uri(instanceDiscoveryEndpoint), requestContext)
-                        .ConfigureAwait(false);
+            var discoveryResponse = await client.DiscoverAadInstanceAsync(new Uri(instanceDiscoveryEndpoint), requestContext)
```

Comment:
>var [](start = 12, length = 3)

Is there a reason to use `var` here and not `InstanceDiscoveryResponse`?

Reply:
no, other than line width.